docs/src/R_comparison.md

```diff
@@ -1,98 +1,121 @@
-# Comparison with R
+# Moving from R to Julia
+This sections presents examples to help move from R to Julia. Examples show R and Julia code for common operations in survey analysis. <br>
+For the same operation, first the R and then the Julia code is presented. 
 
-In the following examples, we'll compare Julia's performance to R's on the same set of operations.
+## Simple random sample
 
-## Installing and loading the package
-**R**
+The `apisrs` data, which is provided in both `survey` and `Survey`, is used as an example. It's a simple random sample of the Academic Performance Index of Californian schools.
 
-```r
-install.package("survey")
+### 1. Creating a design object
+The following example shows how to construct a design object for a simple random sample. 
```

Comment:
Okay, but as a sentence/paragraph, they should make sense, length isn't the only criteria.

Reply:
I agree. They can be longer, length is not necessarily a problem. Repetition is. I am more a fan of stories. We can try to describe the process rather than just say what each line does. This made more sense for the `example.md` file though, which is now part of `index.md`. For the comparison with R it might not be appropriate.

In this case I suggest removing the word "object", from the title as well. It suggests a relation with object-oriented programming, which is not the case. How about "Instantiating a simple random sample survey design"?

Reply:
Can you add committable suggestion like you've done for the others?